DxDispatch/models/fb_bvlcalexnet-12-fp16.json

```diff
@@ -0,0 +1,107 @@
+{
+  "$schema": "./_schema.json",
+
+
+  "resources": 
+  {
+      "data_0": 
+      {
+          "initialValuesDataType": "FLOAT16",
+          "initialValues": { "valueCount": 150528, "value": 0 }
+      },
+      "conv1_w_0": 
+      {
+          "initialValuesDataType": "FLOAT16",
+          "initialValues": { "sourcePath": "B:/Model Files/bvlcalexnet-12-fp16.fb/conv1_w_0.bin" }
+      },
+      "conv1_b_0": 
+      {
+          "initialValuesDataType": "FLOAT16",
+          "initialValues": { "sourcePath": "B:/Model Files/bvlcalexnet-12-fp16.fb/conv1_b_0.bin" }
+      },
+      "conv2_w_0": 
+      {
+          "initialValuesDataType": "FLOAT16",
+          "initialValues": { "sourcePath": "B:/Model Files/bvlcalexnet-12-fp16.fb/conv2_w_0.bin" }
+      },
+      "conv2_b_0": 
+      {
+          "initialValuesDataType": "FLOAT16",
+          "initialValues": { "sourcePath": "B:/Model Files/bvlcalexnet-12-fp16.fb/conv2_b_0.bin" }
+      },
+      "conv3_w_0": 
+      {
+          "initialValuesDataType": "FLOAT16",
+          "initialValues": { "sourcePath": "B:/Model Files/bvlcalexnet-12-fp16.fb/conv3_w_0.bin" }
+      },
+      "conv3_b_0": 
+      {
+          "initialValuesDataType": "FLOAT16",
+          "initialValues": { "sourcePath": "B:/Model Files/bvlcalexnet-12-fp16.fb/conv3_b_0.bin" }
+      },
+      "conv4_w_0": 
+      {
+          "initialValuesDataType": "FLOAT16",
+          "initialValues": { "sourcePath": "B:/Model Files/bvlcalexnet-12-fp16.fb/conv4_w_0.bin" }
+      },
+      "conv4_b_0": 
+      {
+          "initialValuesDataType": "FLOAT16",
+          "initialValues": { "sourcePath": "B:/Model Files/bvlcalexnet-12-fp16.fb/conv4_b_0.bin" }
+      },
+      "conv5_w_0": 
+      {
+          "initialValuesDataType": "FLOAT16",
+          "initialValues": { "sourcePath": "B:/Model Files/bvlcalexnet-12-fp16.fb/conv5_w_0.bin" }
+      },
+      "conv5_b_0": 
+      {
+          "initialValuesDataType": "FLOAT16",
+          "initialValues": { "sourcePath": "B:/Model Files/bvlcalexnet-12-fp16.fb/conv5_b_0.bin" }
+      },
+      "output": {
+          "initialValuesDataType": "FLOAT16",
+          "initialValues": { "valueCount": 9216, "value": 0 }
+      }
+  },
+
+  "dispatchables": 
+  {
+      "bvlcalexnet": 
+      {
+        "type": "dmlSerializedGraph",
+        "sourcePath": "B:/Model Files/bvlcalexnet-12-fp16.fb/Partition_0.bin",
```

Comment:
>bvlcalexnet-12-fp16.fb [](http://example.com/codeflow?start=38&length=22)

The samples should contain the models in the repo and they should get deployed to install\bin\models